### PR TITLE
Refactor common code for creating modules in IncrementalParser.

### PR DESCRIPTION
**Pull request by marsupial**

### CODE REVIEW

File: lib/Interpreter/IncrementalParser.cpp [hunk 1]
```diff
@@ -377,6 +377,18 @@ namespace cling {
     return ParseResultTransaction(T, ParseResult);
   }
 
+  std::string IncrementalParser::makeModuleName() {
+    stdstrstream ModuleName;
+    ModuleName << "cling-module-" << m_ModuleNo++;
+    return std::move(ModuleName.str());
```
Comment (Axel-Naumann) [inline lib/Interpreter/IncrementalParser.cpp]: I'd prefer
```
return std::string("cling-module-") + std::to_string(m_ModuleNo++);
```
Makes sense?

Comment (marsupial) [inline lib/Interpreter/IncrementalParser.cpp]: Yes better.
If they didn't make std::to_string so useless, I wouldn't always forget about it.
Added another commit to remove sstream & iostream includes.

**Comment by marsupial** [comment]
These were done.

**Comment by Axel-Naumann** [comment]
Thanks, merged!

**Pull request closed without merging**